next/frontend/utils/isDefined.ts

```diff
@@ -0,0 +1,3 @@
+export function isDefined<T>(value: T | undefined | null): value is T {
```

Comment:
is it worth to have a separate file for this instead of having it in general.ts ?